client/post-editor/editor-sidebar/feedback-header.jsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import EditorPostType from 'post-editor/editor-post-type';
+
+// TODO: Consider merging this with EditorSidebarHeader depending on final design
+function FeedbackSidebarHeader( { translate, closeFeedback } ) {
+	return (
+		<div className="editor-sidebar__header">
+			<EditorPostType isSettings />
+			<Gridicon icon="arrow-right" />
+			{ translate( 'Share Draft' ) }
+			<Button
+				compact borderless
+				className="editor-sidebar__back"
+				onClick={ closeFeedback }
+				title={ translate( 'Close feedback' ) }
```

Comment:
While it seems pretty far-fetched that this could be taken to mean, "Feedback which is nearby," since it _is_ a translation string and it _is_ ambiguous, we can add a `context` param to let the translators know this is a verb

```js
translate( 'Close feedback', { context: 'verb: imperative' } )
```

Context means that there are multiple ways this could be used in a sentence depending on context and we are describing how to clear it up. Sometimes a string is just a little awkward and needs a little extra nudge to the translators: in these cases we also have `comment`. Each context creates a new translation string but comments only tag along to provide hints, so they are preferred when the string isn't context-sensitive.

```js
translate( 'Approved', { comment: 'indicates thing has been approved blarg blurp' } )
```

See also for `Share Draft` (is it a verb or a noun?)